reference/shipping.v3.yml

```diff
@@ -355,6 +440,41 @@ components:
         - commodity_description
         - international_shipping
         - hs_codes
+    shippingMethods:
+      title: shippingMethods
+      description: Data about the customs information object.
+      type: object
+      x-internal: false
```

Comment:
What does this represent?

Reply:
Let's make it public, I have seen at least a few requests from agencies to have an API to query all methods without having to iterate over all the zones.

Reply:
In that case, it might make sense to add more properties so we have feature parity with V2.